Forge.TreeWalker/src/ForgeDictionary.cs

```diff
@@ -19,11 +19,6 @@ namespace Forge.TreeWalker
     /// </summary>
     public class ForgeDictionary : IForgeDictionary
     {
-        /// <summary>
-        /// The unique identifier for this session.
-        /// </summary>
-        public Guid SessionId { get; set; }
```

Comment:
Is this SessionId not useful for the application? I see that Forge only needs the keyPrefix but the application might find it useful for logging/querying session information etc.

Reply:
oh is it? I was under the impression that during rehydrate scenario, this helps in identifying the subroutine if they are running in paralel?

Reply:
The app can implement IForgeDictionary however they like, including adding additional properties for SessionId if they want access to that directly. I didn't see a need for this property with the addition of UpdateKeyPrefix and GetKeyPrefix. The app could also string parse the GetKeyPrefix result.